### DISCUSSION TITLE
Great Project! Can You Upload It to Unraid Community Appstore or Provide a Custom Install Script?

### DISCUSSION THREAD

**Discussion by JohnnyPh** [WGDashboard/WGDashboard#324] [General]
Hi,

I really like the WGDashboard project; it's an excellent tool with a much nicer interface! I know Unraid already has a built-in WireGuard VPN, but your dashboard adds a lot of value. It would be great if you could consider uploading it to the Unraid Community Appstore, as this would make it much easier for Unraid users to install and manage.

If uploading to the Appstore isn’t possible, could you please provide a script to custom install it on Unraid?

Thanks for your hard work!

**Comment by donaldzou** [comment]
> Hi,
> 
> 
> 
> I really like the WGDashboard project; it's an excellent tool with a much nicer interface! I know Unraid already has a built-in WireGuard VPN, but your dashboard adds a lot of value. It would be great if you could consider uploading it to the Unraid Community Appstore, as this would make it much easier for Unraid users to install and manage.
> 
> 
> 
> If uploading to the Appstore isn’t possible, could you please provide a script to custom install it on Unraid?
> 
> 
> 
> Thanks for your hard work!

Hi! I'm glad you like this project! I'll need to look into this and see how to do it :)
